### PR TITLE
Feature/create or update question integration test

### PR DESCRIPTION
**Pull request by Djoleoo**

### CODE REVIEW

File: Konteh/Konteh.Tests/Konteh.Tests.csproj
Comment (MilosNesh) [inline Konteh/Konteh.Tests/Konteh.Tests.csproj:1]: Do you need this

File: Konteh/back-office-app/.angular/cache/18.2.3/vite/deps/_metadata.json
Comment (MilosNesh) [inline Konteh/back-office-app/.angular/cache/18.2.3/vite/deps/_metadata.json:1]: remove this

File: Konteh/Konteh.Tests/Konteh.Tests.csproj
Comment (Djoleoo) [reply Konteh/Konteh.Tests/Konteh.Tests.csproj:1]: removed

File: Konteh/back-office-app/.angular/cache/18.2.3/vite/deps/_metadata.json
Comment (Djoleoo) [reply Konteh/back-office-app/.angular/cache/18.2.3/vite/deps/_metadata.json:1]: removed

**Review by MilosNesh — APPROVED** [review]

**Pull request merged**